app/assets/javascripts/contacts.js

```diff
@@ -43,27 +45,38 @@ var showSuccess = function(contact, method) {
   );
 };
 
-var erredContacts = function(_xhr, status, error) {
+var recordAndDisplayError = function(xhr, status, error, message) {
   hideFeedback();
-  console.log("retrieveContacts was unsuccessful");
+  console.log(message + " was unsuccessful");
   console.log("Status: " + status);
-  console.log("Error:" + error);
-  showError("Retrieving contacts");
+  console.log("Error:" + JSON.stringify(error));
+  console.log("XHR:" + JSON.stringify(xhr));
+  showError(message);
+};
+
+var erredContacts = function(xhr, status, error) {
+  recordAndDisplayErrorxhr, status, error, "Retrieving contacts");
```

Comment:
Expected an assignment or function call and instead saw an expression.<br>Missing semicolon.<br>Expected an identifier and instead saw ')'.<br>'recordAndDisplayErrorxhr' is not defined.